src/inc/CMakeLists.txt

```diff
@@ -70,6 +70,7 @@ add_library(corguids ${CORGUIDS_SOURCES})
 
 install (FILES cor.h
                corhdr.h
+               corhost.h
```

Comment:
corhost.h is internal implementation detail. We should not be dropping it for packaging. What is it needed for?